views/layout.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Pokenatra</title>
+  <link rel="stylesheet" href="/css/pokestyle.css">
+</head>
+<body>
+  <header>
+    <h1>Pokenatra!</h1>
+  </header>
+<%= yield %>
```

Comment:
could add container div here, around `<%= yield %>`, that way you dont have to add it to individual files like edit.erb and index.erb and so on.